### PR TITLE
 implement TBB

### PR DESCRIPTION
**Pull request by Semyon1104**

### CODE REVIEW

**Comment by Semyon1104** [comment]
> I don't see cmake scripts for tbb (stage - build for onetbb)

just add to cmake script as googletest and link tbb in tests? or build like opencv?

**Comment by allnes** [comment]
After looked at win build, I see we need to use way same as opencv and disable tbb tests

File: 3rdparty/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1,15 @@
...
+    COMMAND ${CMAKE_COMMAND} -S "${CMAKE_SOURCE_DIR}/3rdparty/TBB" -B "${CMAKE_SOURCE_DIR}/3rdparty/TBB/build" -DCMAKE_C_COMPILER_LAUNCHER=${CMAKE_C_COMPILER_LAUNCHER} -DCMAKE_CXX_COMPILER_LAUNCHER=${CMAKE_CXX_COMPILER_LAUNCHER} -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/TBB/build"
+)
+execute_process(
+    COMMAND ${CMAKE_COMMAND} --build "${CMAKE_SOURCE_DIR}/3rdparty/TBB/build" --config "${CMAKE_BUILD_TYPE}"
+    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}/3rdparty/TBB/build"
+)
+
+option(TBB_TEST OFF)
+add_subdirectory(TBB)
```
Comment (aobolensk) [inline 3rdparty/CMakeLists.txt:15]:
```suggestion
add_subdirectory(TBB)

```

File: app/CMakeLists.txt [hunk 2]
```diff
@@ -18,7 +18,7 @@ if(UNIX)
         set(TensorFlow_INCLUDE_DIRS ${tensorflow_INCLUDE_DIR})
         SET(TENSORFLOW_FOUND "TRUE" CACHE INTERNAL "TENSORFLOW_FOUND")
 
-    else()
+        else()
```
Comment (aobolensk) [inline app/CMakeLists.txt:21]:
```suggestion
    else()
```

**Review by aobolensk — CHANGES_REQUESTED** [review]

**Review by aobolensk — APPROVED** [review]

**Pull request merged**